src/aeso_ast_to_fcode.erl

```diff
@@ -681,7 +681,7 @@ split_pat({con, As, I, Pats}) ->
     Xs = [fresh_name() || _ <- Pats],
     {{con, As, I, Xs}, Pats};
 split_pat({tuple, Pats}) ->
-    Xs = [{var, fresh_name()} || _ <- Pats],
+    Xs = [fresh_name() || _ <- Pats],
```

Comment:
I see in the second element of `{tuple, _}` no need for `var` according to the type. Ok.

Reply:
It was confusing a bit because in `fpat` this `var` was mandatory but in `fsplit_pat` no